src/Command/DocsCommand.php

```diff
@@ -26,7 +27,18 @@ protected function configure() {
    * @return int
    */
   protected function execute(InputInterface $input, OutputInterface $output) {
-    $this->localMachineHelper->startBrowser('https://docs.acquia.com/acquia-cli/');
+    $acquia_products = [
+      'Acquia ACLI' => 'https://docs.acquia.com/acquia-cli/',
+      'Acquia CMS' => 'https://docs.acquia.com/acquia-cms/',
+      'Code Studio' => 'https://docs.acquia.com/code-studio/',
+      'Campaign Studio' => 'https://docs.acquia.com/campaign-studio/',
+      'Content Hub' => 'https://docs.acquia.com/contenthub/',
+    ];
+
+    $labels = array_keys($acquia_products);
+    $question = new ChoiceQuestion('Please select the Acquia Product', $labels, $labels[0]);
```

Comment:
I wonder whether we should permit doing something like `acli doc <product>` to not be prompted, similarly to what we've done with e.g. `acli ide:*`

Reply:
We can but I am not sure what value user will enter for a product. Example - for ACMS, user could type acms, acquia_cms etc. We can for sure check the value in existing list and if not there, prompt user to choose. But I feel most of the time it will be prompt (value won't match)

Reply:
Maybe we can create aliases. You mentioned `acms` or `acquia_cms`, why not have both? Maybe I'm overthinking it, but if I'm to use ACLI to open up docs, it'd better have to be blazing fast otherwise I might just go to my browser window instead.

Reply:
Added argument with product alias

Reply:
Lovely, thanks!